build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts

```diff
@@ -81,21 +86,23 @@ const getCodeCoverageSummary = (
 const getCodeCoverageSummaryForPackages = (coverageReport: CodeCoverageComparison): string => {
 	const metrics = codeCoverageDetailsHeader + getMetricRows(coverageReport);
 
-	return `<details><summary><b>${getColorGlyph(coverageReport.branchCoverageDiff)} ${
+	return `<details><summary><b>${getColorGlyph(coverageReport.branchCoverageDiff + coverageReport.lineCoverageDiff)} ${
 		coverageReport.packagePath
-	}:</b> ${formatDiff(coverageReport.branchCoverageDiff)}</summary>${metrics}</table></details>`;
+	}:</b> <br> Line Coverage Change: ${formatDiff(coverageReport.lineCoverageDiff)}  Branch Coverage Change: ${formatDiff(
+		coverageReport.branchCoverageDiff,
+	)}</summary>${metrics}</table></details>`;
 };
 
-const getColorGlyph = (codeCoverageBranchDiff: number): string => {
-	if (codeCoverageBranchDiff === 0) {
-		return '<span style="color: green">■</span>';
+const getColorGlyph = (codeCoverageDiff: number): string => {
+	if (codeCoverageDiff === 0) {
+		return "■";
 	}
 
-	if (codeCoverageBranchDiff > 0) {
-		return '<span style="color: green">⯅</span>';
+	if (codeCoverageDiff > 0) {
+		return "⯅";
 	}
 
-	return '<span style="color: red">⯆</span>';
+	return "<b>⯅</b>";
```

Comment:
What are the b tags for? If you're trying to draw attention to this color is probably a better way -- or maybe two glyphs -- `⯅⯅`